src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -128,7 +128,7 @@ class Microsoft::Terminal::Core::Terminal final :
     void SelectNewRegion(const COORD coordStart, const COORD coordEnd) override;
     const COORD GetSelectionAnchor() const override;
     const std::wstring GetConsoleTitle() const noexcept override;
-    void ColorSelection(const COORD coordSelectionStart, const COORD coordSelectionEnd, const TextAttribute) override;
+    void ColorSelection(const COORD, const COORD, const TextAttribute) override;

```

Comment:
Why did these lose the variable names? I think all the other definitions in this file have variable names. (I know it doesn't matter compiling wise... but what's the justification?)

Reply:
Added back.